apiserver/common/networkingcommon/networkconfigapi.go

```diff
@@ -141,6 +144,7 @@ func (api *NetworkConfigAPI) setOneMachineNetworkConfig(m *state.Machine, networ
 }
 
 func (api *NetworkConfigAPI) getMachineForSettingNetworkConfig(machineTag string) (*state.Machine, error) {
+	logger.Debugf("TAG: %q", machineTag)
```

Comment:
is this still useful to have? Maybe at TRACE? (if you needed  something to understand what was going on, probably *something* is useful to leave around.)

Reply:
My mistake there. I did not intend to include that. I will remove it.